docs/blog/moving-portland-prague-2020-online.rst

```diff
@@ -0,0 +1,52 @@
+.. post:: Apr 29, 2020
+   :tags: portland-2020, prague-2020, conferences, covid-19
+
+Moving our 2020 conferences online
+==================================
+
+Hey everyone,
+
+We hope that you're holding up during these trying times.
+We are all stuck at home, but still thinking about our docs community :)
+
+**We've concluded that it's not possible for us to hold our Portland or Prague conferences in person this year.**
+Safety of the community and everyone else comes first.
+
+**We're planning to hold an online event, with a new format, sometime in August**.
+We will be releasing additional details as we confirm them.
+
+If you currently have a ticket to Write the Docs Portland, your options are:
+
+- pre-order a ticket to the online event for $100 and get the rest refunded.
+- pre-order a ticket to the online event for $100 and contribute some of the rest to sustaining the organization.
+- get a full refund on **May 10th, 2020**. This happen automatically if you don't select either of the previous options.
+
+`Let us know`_ which option you'd like.
+
+We'll be reaching out to all of our sponsors to talk about moving their sponsorship to our online event, and we are excited about offering new sponsorship benefits provided by the new format.
+
+.. _Let us know: http://TODO
+
+Thoughts on the new format
+--------------------------
+
+We're looking at ways to keep the spirit of our events, in an online format, still providing a gathering place for the community.
```

Comment:
```suggestion
We're looking at ways to keep the spirit of our events in a virtual format, so that the event will act as a gathering place for the community.
```